.github/workflows/llm-whisper-evaluation.yml

```diff
@@ -75,7 +75,7 @@ jobs:
             echo "runner=$runner" >> $GITHUB_OUTPUT
 
   llm-whisper-evaluation:
-    # if: ${{ github.event.schedule || github.event.inputs.artifact == 'llm-whisper-evaluation' || github.event.inputs.artifact == 'all' }} # please comment it for PR tests
```

Comment:
revert this change to make this PR more clean.

Reply:
reverted